l3kernel/testfiles/m3expl008.tlg

```diff
@@ -7,6 +7,14 @@ l. ...\ShowDocument
 > \endgroup \@expl@sys@load@backend@@ \begingroup \UseOneTimeHook {begindocument/before}\@kernel@after@begindocument@before \@expl@sys@load@backend@@ .
 \showdocaux ... #2\ignorespaces ->\showtokens {#1}
 l. ...\ShowDocument
+! Package hooks Error: Extra \PopDefaultHookLabel.
+(hooks)                This command will be ignored.
+Type <return> to continue.
+ ...                                              
+l. ...
+LaTeX does not know anything more about this error, sorry.
+Try typing <return> to proceed.
+If that doesn't work, type X <return> to quit.
```

Comment:
This error happens because this test file undefines some file stack commands like `\@expl@push@filename@@`, in a feeble attempt to emulate an older kernel to test reloading `expl3.sty` and `expl3.ltx` (this test was introduced for LaTeX 2020-02-02, when we started preloading `expl3`).  Nowadays it's tougher to emulate a LaTeX without `expl3`, and this test fails for that reason (these reload tests are a nightmare).  For the time being, I disabled the test that tried to load `expl3` in a LaTeX without `expl3` preloaded.

The `.lvt` file does warn you :)
https://github.com/latex3/latex3/blob/c9bba6cfd4765a0a9552307810e813dc4efbdcaf/l3kernel/testfiles/m3expl008.lvt#L15